dev-requirements.in

```diff
@@ -9,6 +9,7 @@ django-extensions
 django-stubs
 djhtml
 honcho
+pip-tools
```

Comment:
Not sure about this one. Because you need `pip-compile` to use the `.in` files, so `pip-tools` is more a requirement, on top everything: python venv + pip-tools and then install all the deps

Reply:
pip-tools is only useful to generate the `requirements.txt`, if you never upgrade the project dependencies, you don't need it at all. You can run the project without pip-tools!

Reply:
Yes you're right. When I arrive the first time I directly use the txt file and so I can install the correct version of pip-tools.
And it also makes the upgrade of pip-tools easier.
:+1: